src/convenience/webhook.ts

```diff
@@ -54,16 +55,28 @@ export interface WebhookOptions {
  */
 export function webhookCallback<C extends Context = Context>(
     bot: Bot<C>,
-    adapter?: SupportedFrameworks | FrameworkAdapter,
```

Comment:
Perhaps we should remove the definition of `SupportedFrameworks` entirely?

Reply:
Yeap.